EIPS/eip-140.md

```diff
@@ -0,0 +1,58 @@
+## Preamble
+
+    EIP: 140
+    Title: REVERT instruction in the Ethereum Virtual Machine
+    Author: Alex Beregszaszi, Nikolai Mushegian (nikolai@nexusdev.us)
```

Comment:
@chriseth do you want to be listed as an author for your paragraphs added in https://github.com/ethereum/EIPs/pull/206/commits/efd2333c806b278b29f483bd5d41d9d7060e37a6 ?

Would that be ok @axic?